lib/disk/modules/QcowDisk.rb

```diff
@@ -252,6 +282,14 @@ def read_entries(n)
     entries
   end
 
+  def refcount_order
+    version < 3 ? 4 : @header['refcount_order']
+  end
+
+  def header_length
+    version < 3 ? 72 : @header['header_length']
```

Comment:
Is this 72 available from the struct definition?  i.e. SIZEOF_QCOW_HEADER_V2 constant up above?  Or is that value different?